src/Command/NewCommand.php

```diff
@@ -52,21 +52,10 @@ protected function execute(InputInterface $input, OutputInterface $output) {
       $dir = Path::makeAbsolute('drupal', getcwd());
     }
 
-    $output->writeln('<info>Creating project. This may take a few minutes</info>');
+    $output->writeln('<info>Creating project. This may take a few minutes.</info>');
     $this->localMachineHelper->checkRequiredBinariesExist(['composer']);
     $this->createProject($project, $dir);
 
-    if (strpos($project, 'acquia/drupal-minimal-project') !== FALSE) {
-      $this->requireDrush($dir);
```

Comment:
Drupal Minimal Project already provides Drush so we don't need to explicitly add it. And since we're no longer modifying dependencies, we don't need to run composer update.

Reply:
ok great.